languages/python/exercises/concept/bools/.docs/instructions.md

```diff
@@ -0,0 +1,41 @@
+In this exercise, you need to translate some rules from the classic game Pac-Man into Elixir functions.
+
+You have four rules to translate, all related to the game states.
+
+> Don't worry about how the arguments are derived, just focus on combining the arguments to return the intended result.
+
+## 1. Define if Pac-Man eats a ghost
+
+Define the `Rules.eat_ghost?/2` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man is able to eat the ghost. The function should return `True` only if Pac-Man has a power pellet active and is touching a ghost.
```

Comment:
```suggestion
Define the `Rules.eat_ghost` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man is able to eat the ghost. The function should return `True` only if Pac-Man has a power pellet active and is touching a ghost.
```

Reply:
This suggestion is better: https://github.com/exercism/v3/pull/1952#discussion_r458059599